sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -1044,7 +1044,7 @@ class InsertSuite extends DataSourceTest with SharedSparkSession {
   test("SPARK-38336 INSERT INTO statements with tables with default columns: negative tests") {
     object Errors {
       val COMMON_SUBSTRING = " has a DEFAULT value"
-      val COLUMN_DEFAULT_NOT_FOUND = "`default` cannot be resolved."
+      val COLUMN_DEFAULT_NOT_FOUND = "The table or view `t` cannot be found"
```

Comment:
The error message doesn't seems related to COLUMN_DEFAULT_NOT_FOUND

Reply:
Fixed.